django_saml2_auth/tests/test_user.py

```diff
@@ -0,0 +1,189 @@
+import pytest
+from django_saml2_auth.user import *
```

Comment:
This kills me, sorry 😅

Reply:
I intentionally used wildcard, because I use modules imported in that `django_saml2_auth.user`. But I'll change them. :slightly_smiling_face:

Reply:
Done! :heavy_check_mark: